docs/src/pages/docs/crocks/Maybe.md

```diff
@@ -1053,6 +1053,64 @@ safe(isSmall, [ 1, 2 ])
 //=> Just [ 1, 2 ]
 ```
 
+#### safeAfter
+
+`crocks/Maybe/safeAfter`
+
+```haskell
+safeAfter :: ((b -> Boolean) | Pred) -> (a -> b) -> a -> Maybe b
+```
+
+Often times, you might want to lift the result of a function into a `Maybe` based
+on some predicate. Similar to [`safe`](#safe), you pass either a `Pred` or a 
+predicate function, along with a unary function. `safeAfter` then gives you a new function, 
+which when invoked evaluates the predicate against the result of executing 
+the unary function. This result is then lifted into a `Just` if the predicate evaluates 
+to `true`, or a `Nothing` if it returns `false`
+
+```javascript
+import Pred from 'crocks/Pred'
```

Comment:
This example does not match the example in the README and things in the docs structure are not linted (right now), so there is a chance the code may contain linting errors that go unnoticed. What I typically do is write the README first, and then copy it over to the docs folder when the README passes the linter.